addons/html_editor/tests/test_controller.py

```diff
@@ -182,18 +182,46 @@ def test_05_internal_link_preview(self):
         self.assertEqual(200, response_wrong_record.status_code)
         self.assertTrue('error_msg' in response_wrong_record.text)
 
-        # retrieve metadata of a url not directing to a record
-        response_not_record = self.url_open(
+        # retrieve metadata of a url directing to a frontend page (not a record)
+        # page without seo description set
+        view = self.env["ir.ui.view"].create({
+            "arch": """
+            <t t-name="website.my-page">
+                <h1>My Page</h1>
+            </t>""",
+            "type": "qweb"
+        })
+        self.env["website.page"].create({
+            "url": "/my-page",
+            "view_id": view.id,
+        })
+        response_frontend_page_no_seo = self.url_open(
```

Comment:
after adapting the suggestion, pls remember to change the test acordingly, we won't need to initialize a `website.page` env